Updated Version for 2024

I-T-C-R-W opened the pull request:
changed the design just a bit 
and updated to newer Versions

I-T-C-R-W committed:
Add files via upload

NEW VERSION FOR 2024

I-T-C-R-W committed:
Update README.md

I-T-C-R-W committed:
Delete PyQt Circular Colorpicker.py

I-T-C-R-W commented:
enjoy the newer version

I-T-C-R-W committed:
Delete demo directory

I-T-C-R-W committed:
Update README.md

omega0verride commented:
Hi @ I-T-C-R-W I don't know if we want to merge this, I noticed that there are some changes in the UI design. I had some extra time so I upgraded the source to PyQt6. In addition to that I did some major code improvements as the initial implementation was done years ago while I was still tinkering with QT.

Maybe we can add a smaller text that displays the hex code, the default colors can be pushed a bit lower and we can add a right-click menu that allows copying the color code in different formats (rgb, rgba, hsl, hex etc.). We could also enable pasting hex and rgb codes other formats can be pasted through the context menu.
![image](https://github.com/user-attachments/assets/103278b8-24fd-4a44-8363-c5d8e40d3852)

I-T-C-R-W committed:
Update pyqtpicker.py

v040_stable 

- fixed a working dir error if used as instance
- fixed use of Hexcode 
- new readme with code example for use

I-T-C-R-W committed:
Add files via upload

I-T-C-R-W commented:
Hey there,  
i have changed it a for external use; 
so i can fetch the code per button click in my main app if the picker is open. 
(see Readme)      - fixed some small problems of my last Version.    
For my Theme-maker this is ok now and very easy to use.  So i do not need any more features. If you dont want to merge - thats fine of me ,too.   
Thank you for your work ! - which was a perfect startingpoint for the Version i use now. 

Best regards

I-T-C-R-W committed:
Update README.md

I-T-C-R-W committed:
Update README.md

I-T-C-R-W committed:
Update README.md
